ext/span.c

```diff
@@ -184,12 +184,13 @@ void ddtrace_open_span(ddtrace_span_data *span) {
     ddtrace_set_global_span_properties(span);
 }
 
+// += 2 increment to avoid zval type ever being 0
```

Comment:
That looks like magic :)

Reply:
Yes, I don't like it very much, but it works.